cmo /update-monitoring-plugin-image.yaml

```diff
@@ -0,0 +1,21 @@
+# 1) scale down Cluster Version Operator deployment to 0
+
+oc scale --replicas=0 -n openshift-cluster-version deployment/cluster-version-operator
+
+# 2) Set Cluster Monitoring Operator (CMO) to unmanged 
+# Resource https://access.redhat.com/solutions/6548111
+
+oc patch clusterversion version --type json -p "$(cat disable-monitoring.yaml)"
+
+oc scale --replicas=0 -n openshift-monitoring deployment/cluster-monitoring-operator
```

Comment:
I'm not sure if the operator itself needs to be scaled down, this might cause some other issues. Wouldn't be enough to out it into unmanaged?